### PR TITLE
Project Properties Updated

### PR DESCRIPTION
**Pull request by BRyeGmoney**
-added address fields present in v1.8.0 of the PlanGrid API

---
This change is [Reviewable](https://reviewable.io/reviews/plangrid/plangrid-api-net/10)

### CODE REVIEW

File: PlanGrid.Api/Project.cs [hunk 1]
```diff
@@ -10,5 +10,41 @@ public class Project : Record
...
+        public string Type { get; set; }
+
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("owner")]
+        public string Owner { get; set; }
+
+        [JsonProperty("start_date")]
+        public DateTime? StartDate { get; set; }
```
Comment (JuliusAlexanderIV) [inline PlanGrid.Api/Project.cs]: `StartDate` and `EndDate` should be `Date?`s for both classes as they're ISO dates and are nullable. Check out the [project object](https://developer.plangrid.com/docs/the-project-object) to see their format.

**Comment by JuliusAlexanderIV** [comment]
@BRyeGmoney looks good, thanks for updating this! I just have one comment.

File: PlanGrid.Api/Project.cs (hunk 1 above)
Comment (kswoll) [inline PlanGrid.Api/Project.cs]: Ah, good point.  So they never include a time component?  In that case, yeah, @JuliusAlexanderIV is right and they should use our custom `Date` class instead.  Also, I second my thanks! :)

**Comment by BRyeGmoney** [comment]
@JuliusAlexanderIV @kswoll no problemo. i imagine you guys are doing the squashing as part of the merge?

**Comment by JuliusAlexanderIV** [comment]
@BRyeGmoney thanks for the PR, I just merged it into the master branch so it's now available on Nuget!

**Pull request closed without merging**